src/main/webapp/student/enrollment/schoolClass/schoolClassesSelection.jsp

```diff
@@ -32,8 +32,13 @@
 <h1><bean:message bundle="STUDENT_RESOURCES" key="title.student.shift.enrollment" /></h1>
 
 <c:if test="${not empty enrollmentBeans}">
-	<div class="alert alert-warning" role="alert"><bean:message bundle="STUDENT_RESOURCES" key="message.warning.student.enrolmentClasses" /> 
-		<html:link page="<%= "/studentEnrollmentManagement.do?method=prepare" %>" styleClass="alert-link" style="color: #7F3C00"><bean:message bundle="STUDENT_RESOURCES" key="message.warning.student.enrolmentClasses.Fenix" /></html:link>.
+	<div class="alert alert-warning" role="alert"><bean:message bundle="STUDENT_RESOURCES" key="message.warning.student.enrolmentClasses" />
+		<c:if test="${not empty workflowRegistrationOid}">		 
+			<bean:message bundle="STUDENT_RESOURCES" key="message.warning.student.enrolmentClasses.Fenix" />.
```

Comment:
@NunoPinheiro if in the middle of workflow process, maybe this warning message isn't necessary at all...